documents/views.py

```diff
@@ -88,17 +88,9 @@ def document_edit(request, pk):
         if settings.READ_ONLY:
             return HttpResponse("Upload is disabled for a few hours", status=401)
 
-        form = FileForm(request.POST)
-
+        form = DocumentForm(request.POST, instance=doc)
         if form.is_valid():
-            doc.name = form.cleaned_data["name"]
-            doc.description = form.cleaned_data["description"]
-
-            doc.tags.clear()
-            for tag in form.cleaned_data["tags"]:
-                doc.tags.add(Tag.objects.get(name=tag))
-
-            doc.save()
+            new_doc = form.save()
```

Comment:
It's still the same document
```suggestion
            doc = form.save()
```

Reply:
should actually just be form.save()